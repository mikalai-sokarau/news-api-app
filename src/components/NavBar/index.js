import React from "react";
import { Link } from "react-router-dom";

const NavBar = () => (
  <div className="navbar-fixed">
    <nav>
      <div className="nav-wrapper indigo lighten-4 row">
        <div className="col s9">
          <Link to="/" className="bran-logo-center">
            MY FEED
          </Link>
        </div>
        <div className="col s3">
          <label>
            <input type="checkbox" className="filled-in" />
            <span>Show favorite news</span>
          </label>
        </div>
      </div>
    </nav>
  </div>
);

export default NavBar;
