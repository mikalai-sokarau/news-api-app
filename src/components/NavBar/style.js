import styled from "styled-components";

const StyledDiv = styled.div`
  display: flex;
  align-items: center;
  justify-content: flex-end;
`;

const StyledContainer = styled.nav`
  display: flex;
`;

export { StyledDiv, StyledContainer };
