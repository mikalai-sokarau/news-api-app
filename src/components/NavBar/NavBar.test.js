import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './index';

describe('<NavBar/>', () => {
  test('className should be `navbar-fixed`', () => {
    const wrapper = shallow(<NavBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
