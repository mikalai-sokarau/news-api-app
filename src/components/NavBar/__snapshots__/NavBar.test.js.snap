// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar/> className should be \`navbar-fixed\` 1`] = `
<div
  className="navbar-fixed"
>
  <nav>
    <styled.nav
      className="indigo lighten-4 row"
    >
      <div
        className="col s10"
      >
        <ul>
          <li>
            <Link
              className="bran-logof-center"
              replace={false}
              to="/"
            >
              MY FEED
            </Link>
            |
          </li>
          <li>
            <Link
              replace={false}
              to="/images"
            >
               IMAGES 
            </Link>
            |
          </li>
          <li>
            <Link
              replace={false}
              to="/map"
            >
              MAP
            </Link>
          </li>
        </ul>
      </div>
      <styled.div
        className="col s2"
      >
        <Button />
      </styled.div>
    </styled.nav>
  </nav>
</div>
`;
